pkg/frontend/load.go

```diff
@@ -455,8 +455,8 @@ func collectWriteBatchResult(handler *ParseLineHandler, wh *WriteBatchHandler, e
 	wh.simdCsvErr = nil
 }
 
-func makeParsedFailedError(tp, field, column string, line uint64, offset int) *MysqlError {
-	return NewMysqlError(ER_TRUNCATED_WRONG_VALUE_FOR_FIELD,
+func makeParsedFailedError(tp, field, column string, line uint64, offset int) *moerr.MysqlError {
+	return moerr.NewMysqlError(moerr.ER_TRUNCATED_WRONG_VALUE_FOR_FIELD,
```

Comment:
should we just use `moerr.New(...)` ?

Reply:
no need to keep `moerr.NewMysqlError`, is that right?

Reply:
done